src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -22,8 +22,8 @@ private void TestWindowApisUnix()
             // This tests the hardcoded implementations of these APIs on Unix.
             using (Process p = Process.GetCurrentProcess())
             {
-                Assert.True(p.Responding);
-                Assert.Equal(string.Empty, p.MainWindowTitle);
+                Assert.Throws<PlatformNotSupportedException>(() => p.Responding);
```

Comment:
@stephentoub @danmosemsft - ok will change that for both uap and linux - intuitively: Responding, NotResponding, NoClueBecauseIDontKnowHowToDoThatOnLinuxOrUAP makes more sense to me though